src/Command/ApplyCommand.php

```diff
@@ -32,6 +33,16 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $config = $serializer->deserialize(file_get_contents('shipyard.yaml'), ShipyardModel::class, 'yaml');
 
+        $tag = $input->getOption('tag') ? $input->getOption('tag') : False;
+        $settings = $config->getSettings();
+        
+        if($tag) {
+            $settings->setShipyardTag($tag);
+        } else {
+            $settings->setShipyardTag(false);
+        }
+        $config->setSettings($settings);
+
```

Comment:
Hi Kostiantyn,

Here's a improvement suggestion. This should have the same effect. I might be mistaken, so please verify if you decide to apply it.

```suggestion
        $settings->setShipyardTag($tag);
```

Here's my reason:
* **The `if` removal**:  In case $tag been false we can set it to `$settings->setShipyardTag($tag);` the same way.
* **The `$config->setSettings($settings);` removal**: `$settings` is a mutable object, therefore a reference and not a copy, meaning that `$settings === $config->getSettings()`.


BR,